skedulr/app/models/business.rb

```diff
@@ -1,3 +1,11 @@
 class Business < ApplicationRecord
   has_many :employees
+
```

Comment:
Could you add a before_save here to lower case the business name before it's put into the database?